SSCCSET2019/SSCCSET2019/Logic/ActivityPageLogic.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+using SSCCSET2019.Pages.HomePage;
+using SSCCSET2019.Tools.Driver;
+
+namespace SSCCSET2019.Logic
+{
+    class ActivityPageLogic
+    {
+        IWebDriver driver = Driver.GetDriver();
+        public void MakeReply(string replyText) 
+        {
+            Activity act = new Activity(driver);
```

Comment:
Plese, change return type for the methods in the page.

Reply:
did it